File: 1962.py
# Python 3.4.3

def get_ano(anos_transcorridos, ano):
  diferenca = ano - anos_transcorridos

  if diferenca == 0:
    return '1 A.C.'

  if diferenca > 0:
    return '%d D.C.' % (diferenca)

  return '%d A.C.' % (diferenca*-1 + 1)

def main():
  n = int(input())

  for i in range(n):
    anos_transcorridos = int(input())

    print(get_ano(anos_transcorridos, 2015))

if __name__ == '__main__':
  main()

File: 1962.js
// Node.js 12.18.3

function getAno(anosTranscorridos, ano) {
  const diferenca = ano - anosTranscorridos;

  if(diferenca === 0) {
    return '1 A.C.';
  }

  if(diferenca > 0) {
    return `${diferenca} D.C.`;
  }

  return `${diferenca*-1 +1} A.C.`;
}

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

lines.shift();

for (let line of lines) {
  if(!line) { // prevent reading an empty line
    continue;
  }
  const anosTranscorridos = Number(line);
  console.log(getAno(anosTranscorridos, 2015))
}